### PR TITLE
v3benchmark: add put benchmark

### PR DESCRIPTION
**Pull request by xiang90**
/cc @yichengq

### CODE REVIEW

File: tools/v3benchmark/main.go [hunk 1]
```diff
@@ -62,22 +69,21 @@ func main() {
...
+	if act == "get" {
+		var rangeEnd []byte
+		key := []byte(flag.Args()[1])
+		if len(flag.Args()) > 2 {
+			rangeEnd = []byte(flag.Args()[2])
+		}
+		benchGet(conn, key, rangeEnd, n, c)
+	} else if act == "put" {
+		key := []byte(flag.Args()[1])
+		kc, err := strconv.ParseInt(flag.Args()[2], 10, 32)
```
Comment (yichengq) [inline tools/v3benchmark/main.go]: what is kc? could we add a comment for it?

**Comment by yichengq** [comment]
lgtm

File: tools/v3benchmark/main.go (hunk 1 above)
Comment (xiang90) [inline tools/v3benchmark/main.go]: fixed.

Comment (yichengq) [inline tools/v3benchmark/main.go]: :ok:

**Pull request merged**